lib/lm3s/rcc.c

```diff
@@ -0,0 +1,105 @@
+/** @defgroup rcc_file RCC Controller
+
+@brief <b>LM3S RCC Controller</b>
+
+@ingroup LM3Sxx
+
+@version 1.0.0
+
+@author @htmlonly &copy; @endhtmlonly 2011
+Daniele Lacamera <root at danielinux dot net>
+
+@date 21 November 2015
+
+LGPL License Terms @ref lgpl_license
+*/
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2009 Uwe Hermann <uwe@hermann-uwe.de>
+ * Copyright (C) 2009 Federico Ruiz-Ugalde <memeruiz at gmail dot com>
+ * Copyright (C) 2011 Fergus Noble <fergusnoble@gmail.com>
+ * Copyright (C) 2011 Stephen Caudle <scaudle@doceme.com>
+ * Copyright (C) 2015 Daniele Lacamera <root@danielinux.net>
```

Comment:
as elsewhere, I'm hard pressed to believe there's anything of _any_ relevance from any other authors here, certainly not as far back as 2009.